### PR TITLE
return OK when parser error happens or other unrecognized requests r…

### PR DESCRIPTION
**Pull request by cothee**
1. return OK when parser error happens or other unrecognized requests reach;
2. support stall_status command in the client;
3. force to parse table header in each client request but not each client connection

### CODE REVIEW

File: src/pg_conn.cc [hunk 1]
```diff
@@ -308,6 +317,10 @@ bool PGConn::Glog(const std::string &msg) {
   return true;
 }
 
+void PGConn::StallStatus(std::string &ss) const {
```
Comment (gaodq) [inline src/pg_conn.cc:320]: 这个函数什么作用？

File: src/pg_conn.cc [hunk 2]
```diff
@@ -185,15 +185,21 @@ ReadStatus PGConn::HandleNormal() {
         // for lavavel PDO's last phase.
         AppendCommandComplete();
         return kReadAll;
-      } else if (parser_.Parse(error_info)) {
+      } else if (statement_.find("stall_status") != std::string::npos) {
+          AppendStatus();
+          return kReadAll;
+        }else if (parser_.Parse(error_info)) {
```
Comment (gaodq) [inline src/pg_conn.cc]: else前加个空格吧

File: src/pg_conn.cc (hunk 1 above)
Comment (cothee) [reply src/pg_conn.cc:320]: 在PGConn里面具体实现为空，在gpstall那边的子类里面就是负责构造gpstall的运行状态信息。

File: src/pg_conn.cc (hunk 2 above)
Comment (cothee) [reply src/pg_conn.cc]: OK

**Pull request merged**